src/parser/apiSpec.yaml

```diff
@@ -0,0 +1,126 @@
+swagger: "2.0"
+info:
+  description: "API description for the Parser/Database component of the Road Search Engine."
```

Comment:
I don't think the description should be self aware that it's a description. I'd just remove the word "description"